mongodb-osx-x86_64-3.4.0/MPL-2

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
What is the mongo-db-osx doing in your codebase? Nothing but your own code should be in your repo.